src/components/Tooltip/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -30,12 +30,12 @@ exports[`Tooltip render variant white 1`] = `
   -ms-transform: translate3d(0, -4px, 0);
   transform: translate3d(0, -4px, 0);
   background-color: #fff;
-  color: #000;
```

Comment:
Strange, here it was white before

Reply:
This is for the white variant so the text should be black, maybe we should use `neutral.textStrong` and set the contrast of `textStrong` to 1000?

Reply:
not white: black -> https://www.color-hex.com/color/4a4f62